docs/business-apps/power-automate/get-started/create-your first-flow.md

```diff
@@ -49,16 +46,16 @@ Before proceeding with these instructions, make sure your SharePoint site is set
 
     ![Flow template - send email when an item is added in a list](../../../images/gs01-create-a-flow-when-item-is-added-template.png)
 
-6. To create the flow, select **Create Flow**. Any connections that require a connection is also attempted during this step. After the flow is created successfully, you are redirected to the flow designer where you can edit and modifify the flow if needed.
+6. To create the flow, select **Create Flow**. Any connections that require a connection is also attempted during this step. After the flow is created successfully, you are redirected to the flow details page where you can edit and modifify the flow if needed.
+    >Your flow is active as soon as you save the flow. In this case, this flow that will send an email when new items are added to the list is now active, and runs whenever new items are added to the **Projects** list in the specified SharePoint site.
```

Comment:
In this case, this flow that sends an email when new items are added to the list is now active, and runs whenever new items are added to the **Projects** list in the specified SharePoint site.